{{ cookiecutter.repo_name }}/requirements.txt

```diff
@@ -0,0 +1 @@
+spyder==3.1.3
```

Comment:
But dependencies will updated using pyup, and exact pinning avoid them to be automatically updated.